csc2/macc.h

```diff
@@ -75,7 +75,7 @@ extern char *maccfuncpath;
 
 /* MAXINDEX HAS BECOME OBSOLETE - FOR NOW, JUST KLUDGE TO FIX */
 #define MAXKEYS 256  /* maximum # of keys with cases */
-#define MAXINDEX 256 /* max # of indices */
+//#define MAXINDEX 256 /* max # of indices */
```

Comment:
Do we really need this here? For now, I have commented it in favor of one defined in bbinc/cdb2_constants.h (the value of which is 50 though).